src/helpers/customers.js

```diff
@@ -187,6 +193,30 @@ exports.createCustomer = async (payload, credentials) => {
   return newCustomer;
 };
 
+exports.removeCustomer = async (customerId) => {
+  const customer = await Customer.findOne({ _id: customerId }, { driveFolder: 1 }).lean();
+  const helper = await Helper.findOne({ customer: customerId }, { user: 1 }).lean();
```

Comment:
je suis désolée je n'avais pas vu : un bénéficiaire peut avoir plusieurs aidants donc ici il faut que tu mettes `find` et en dessous il faut que tu fasses un boucle for

Reply:
oh oui bien vu, j'aurai dû y penser !

Reply:
done